### ISSUE TITLE
add tidyselect dependency to manipulate validation file ordering

### ISSUE THREAD

**Issue by mariev** [phuse-org/valtools#67]
`vt_add_file_to_config`  allows insertion of 1+ new filenames before/after an explicitly named existing file in the list. Extend this to use tidyselect verbs for true reordering, per discussion: https://github.com/phuse-org/valtools/pull/57/files#r592896357

**Comment by thebioengineer** [comment]
Out of curiosity, I had a look at the tidyselect package and dplyr::relocate. It should be straightforward to drop it in with what you wrote already! 👍 

more or less copied from dplyr::relocate: 

```r
tidy_string_select <- function(input, .before = NULL, .after = NULL, pkg = "."){
  
  .before <- enquo(.before)
  .after <- enquo(.after)
  
  .data <- get_config_validation_files(dir = pkg)
  .data <- setNames(.data, .data)
  
  has_before <- !quo_is_null(.before)
  has_after <- !quo_is_null(.after)
  if (has_before && has_after) {
    abort("Must supply only one of `.before` and `.after`.")
  } else if (has_before) {
    where <- min(unname(tidyselect::eval_select(.before, .data)))
  } else if (has_after) {
    where <- max(unname(tidyselect::eval_select(.after, .data)))
  } else {
    where <- length(.data) + 1
  }
  
  lhs <- .data[seq2(1, where - 1)]
  output <- unique(c(lhs, input))
  
  if(where-1 != length(.data) ){
    rhs <- .data[seq2(where + 1, length(.data))]
    output <- unique(output, rhs)
  }
  return(output)
}
```

**Comment by thebioengineer** [comment]
Resolved with PRs #109 and #98
